load/PreValidate.py

```diff
@@ -48,13 +67,11 @@ def validateLambda(lptsReader, directory, filenames):
 		result = None
 		stockNumber = preValidate.isTextStockNo(directory)
 		if stockNumber != None:
-			resultList = preValidate.validateUSXStockNo(stockNumber, filenames)
-			if resultList != None and len(resultList) > 0:
-				result = resultList[0]
+			stockResultList = preValidate.validateUSXStockList(stocknumberArrayFinal, filenames, sampleText)
```

Comment:
the call to validateLambda shouldn't have sampleText. that is only for validate DBPETL

Reply:
@bradflood, It has removed the sampleText parameter when validateLambda method is called.